src/google.py

```diff
@@ -7,11 +10,52 @@
 from src.constants.http_status_codes import HTTP_200_OK,HTTP_201_CREATED, HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND
 from src.models import UserLogin, UserProfile, db, Borrow
 
+load_dotenv()
+
 
 # Blueprint for google module.
 google_bp = Blueprint('google', __name__, url_prefix='/api')
 
 
+
+def validate_issuer(iss):
+    """Validate the Issuer of the id_token.
+
+    Args:
+        aud (string): Token Issuer
+
+    Returns:
+        bool: True\False. True if the token issuer is correct.
+    """
+    return True if os.environ.get('TOKEN_ISSUER') == iss else False
+
+
+def validate_client(aud):
+    """Validate the Client's ID on the token.
+
+    Args:
+        aud (string): Audience ID
+
+    Returns:
+        bool: True\False. True if the client is correct.
+    """
+    return True if os.environ.get('CLIENT_ID') == aud else False
+
+
+def token_expired(expire):
+    """Checks if a google tone is expired. Returns
+
+    Args:
+        expire (int): expiration time in unix epoch
+        
+    Returns:
+        bool: False | True. Returns "False" if token has not expired, otherwise True
+    """
+    exp = int(expire)
+    now = int(time.time())
+    return True if now > exp else False
```

Comment:
You should be able to compare time values in python without converting to integer. Also, you can still do this in the place where it is being used instead of making a whole new function for it.

Reply:
It's an integer value of the [Unix Epoch time](https://www.epoch101.com/) from the token payload, `"exp": 1640703916`. Hence it make sense to compare integers.